noir-projects/noir-protocol-circuits/crates/types/src/address/aztec_address.nr

```diff
@@ -59,25 +59,6 @@ impl AztecAddress {
         )
     }
 
-    pub fn compute_from_public_keys_and_partial_address(
```

Comment:
Nuked this as it was only used in 2 places and it felt a bit redundant to `compute(...)`